mq/publishers.go

```diff
@@ -134,6 +148,101 @@ func NodeUpdate(node *models.Node) error {
 	return nil
 }
 
+func ServerStatusUpdate() error {
```

Comment:
why do we need this functionality?
status API is not costly, having these operations over mq might be an overkill

Reply:
In mq approach, server only needs to send one message(one connection, rather than every client api call).
There is also another func to check if there is server status change. Only the server status is changed, a new mq message is distributed to clients.   It will save a lot of api requests.